### PR TITLE
Initial conda recipe for lume-genesis

### PR DESCRIPTION
**Pull request by balticfish**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by balticfish** [comment]
@conda-forge/help-python this is ready for review

**Comment by ChristopherMayes** [comment]
@conda-forge/help-python this is ready for review

**Comment by ChristopherMayes** [comment]
@conda-forge/help-python I also agree to be a maintainer.

File: recipes/lume-genesis/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - bokeh
+    - matplotlib
```
Comment (scopatz) [inline recipes/lume-genesis/meta.yaml]: Is it OK for this to be `matplotlib-base` instead? Or do you really need the Qt backend

Comment (balticfish) [reply recipes/lume-genesis/meta.yaml]: Yes, it can be matplotlib-base.

Comment (balticfish) [reply recipes/lume-genesis/meta.yaml]: @scopatz I have corrected it.

**Comment by balticfish** [comment]
@conda-forge/help-python this is ready for review

**Comment by npavlovikj** [comment]
Thank you for contributing @balticfish 🎉

**Pull request merged**